releng/org.eclipse.xtext.target/org.eclipse.xtext.target-r202103.target

```diff
@@ -0,0 +1,110 @@
+	</location>
+	
+	<location includeAllPlatforms="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+		<unit id="org.eclipse.xtext.builder.standalone" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.builder.standalone.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.common.types" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.common.types.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.ecore" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.ecore.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.java" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.java.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.purexbase" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.purexbase.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.purexbase.ide" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.purexbase.ide.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.smap" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.smap.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.ide" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.ide.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.testing" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.testing.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.testlanguages" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.testlanguages.source" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.testlanguages.ide" version="0.0.0"/>
+		<unit id="org.eclipse.xtext.xbase.testlanguages.ide.source" version="0.0.0"/>
+		<repository location="https://ci.eclipse.org/xtext/job/xtext-extras/job/master/lastStableBuild/artifact/build/p2-repository/"/>
+	</location>
+	
+	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+		<unit id="org.eclipse.platform.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.jdt.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.pde.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.pde.api.tools.ee.feature.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.draw2d.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.emf.mwe2.lib" version="0.0.0"/>
+		<unit id="org.eclipse.emf.mwe2.launcher.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.m2e.feature.feature.group" version="0.0.0"/>
+		<unit id="org.eclipse.buildship.feature.group" version="0.0.0"/>
+		<repository location="https://download.eclipse.org/releases/2021-03"/>
+	</location>
+
+	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+		<unit id="org.eclipse.emf.sdk.feature.group" version="0.0.0"/>
+		<repository location="https://download.eclipse.org/modeling/emf/emf/builds/release/2.25"/>
```

Comment:
https://download.eclipse.org/modeling/emf/emf/builds/release/2.24

Reply:
yes should be released soon

Reply:
ok now